events/README.md

```diff
@@ -14,6 +14,8 @@ This package provides input types for Lambda functions that process AWS events.
 
 [CloudFormation Events](../cfn/README.md)
 
+[Chime Bot Events](README_Chime_Bots.md)
```

Comment:
This should probably be "Chime Bot Event" (without the trailing "s")

Reply:
There is an `EventType` field, plural makes sense to me :)